multigeiger/tube.cpp

```diff
@@ -208,10 +208,8 @@ void setup_tube(void) {
   digitalWrite(PIN_TEST_OUTPUT, LOW);
   digitalWrite(PIN_HV_FET_OUTPUT, LOW);
 
-  unsigned long now_ms = millis();
-
   // note: we do not need to get the portMUX here as we did not yet enable interrupts.
-  isr_count_timestamp = now_ms;
+  isr_count_timestamp = 0;
```

Comment:
this gets overwritten at the first GM pulse.

before that, we better have something we can detect easily.